metadata.rb

```diff
@@ -5,13 +5,13 @@
 description 'Upgrades chef-client to specified releases'
 version '3.12.0'
 
-chef_version '>= 11' if respond_to?(:chef_version) # cookstyle: disable ChefModernize/RespondToInMetadata
+chef_version '>= 12.5' if respond_to?(:chef_version) # cookstyle: disable Chef/Modernize/RespondToInMetadata
 
 %w(amazon centos debian mac_os_x opensuseleap oracle redhat scientific solaris2 suse ubuntu windows aix).each do |os|
   supports os
 end
 
-unless defined?(Ridley) # cookstyle: disable ChefModernize/RespondToInMetadata
-  source_url 'https://github.com/chef-cookbooks/chef_client_updater' if respond_to?(:source_url) # cookstyle: disable ChefModernize/RespondToInMetadata
-  issues_url 'https://github.com/chef-cookbooks/chef_client_updater/issues' if respond_to?(:issues_url) # cookstyle: disable ChefModernize/RespondToInMetadata
+unless defined?(Ridley) # cookstyle: disable Chef/Modernize/RespondToInMetadata
```

Comment:
I think the unless Ridley was a chef 11 thing that can go away but @lamont-granquist would know best here